homeassistant/components/tuya/button.py

```diff
@@ -48,6 +48,38 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    DeviceCategory.MSP: (
+        ButtonEntityDescription(
+            key=DPCode.EMPTY,
+            translation_key="empty",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        ButtonEntityDescription(
+            key=DPCode.FACTORY_RESET,
+            translation_key="factory_reset",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        ButtonEntityDescription(
+            key=DPCode.CLEAN,
+            translation_key="manual_clean",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        ButtonEntityDescription(
+            key=DPCode.MANUAL_CLEAN,
+            translation_key="manual_clean",
+            entity_category=EntityCategory.CONFIG,
+        ),
```

Comment:
Can you add a comment to indicate why we have two entities with different key but same translation key?